### DISCUSSION TITLE
"Upload a photo" Button Not Functional in Profile Picture Settings

### DISCUSSION THREAD

**Discussion by haleyjung** [community/community#113693] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

**Bug Description:**
When navigating to Settings > Profile Picture > Edit and attempting to upload a new photo by clicking the "Upload a photo" button, the expected action does not occur. Clicking the button does not prompt the user to select an image for uploading as it should.

**Steps to Reproduce:**
1. Log in to the account.
2. Navigate to Settings.
3. Select Profile Picture.
4. Click on the Edit option.
5. Attempt to upload a new photo by clicking the "Upload a photo" button.

**Expected Behavior:**
Clicking the "Upload a photo" button should prompt the user to select an image file for uploading as a new profile picture.

**Actual Behavior:**
Upon clicking the "Upload a photo" button, no action is triggered, and the option to select an image does not appear.

**Additional Information:**
Removing a profile image still functions correctly.

**Comment by haleyjung** [comment]
Issue solved

**Reply by soyoonmoon** [reply]
Are you sure? The button is not working right now

**Reply by haleyjung** [reply]
I just retried and it still works for me. Are you facing the same issue with the upload button?
